Revise prerequisites for dbt service token creation to include PAT token (#8736)
fishtown-analytics/docs.getdbt.com: website/docs/guides/airflow-and-dbt-cloud.md

## Prerequisites

- [<Constant name="cloud" /> Enterprise or Enterprise+ account](https://www.getdbt.com/pricing/)

> - Service Account token if you have [IT license (with admins access)](/docs/cloud/manage-access/enterprise-permissions)). Permissions for service tokens can be found [here](/docs/dbt-cloud-apis/service-tokens#permissions-for-service-account-tokens).

Reviewer: ```suggestion
- Service account token if you have [Developer license (with admins access)](/docs/cloud/manage-access/enterprise-permissions)). Permissions for service tokens can be found [here](/docs/dbt-cloud-apis/service-tokens#permissions-for-service-account-tokens) and [here](/docs/cloud/manage-access/seats-and-users?version=2.0).
```

> - Service Account token if you have [IT license (with admins access)](/docs/cloud/manage-access/enterprise-permissions)). Permissions for service tokens can be found [here](/docs/dbt-cloud-apis/service-tokens#permissions-for-service-account-tokens).

Reviewer: hey @evil-in thanks so much for opening this pr up! curious if you've experienced creating a service account token with an IT license w admin access? The [docs](https://docs.getdbt.com/docs/cloud/manage-access/seats-and-users?version=2.0) currently say only dev licenses can create a service token. we'd love to look into this more

> - [PAT Token](https://docs.getdbt.com/docs/dbt-cloud-apis/user-tokens#create-a-personal-access-token) if you have developer license.

Reviewer: ```suggestion
- [Personal access token (PAT)](https://docs.getdbt.com/docs/dbt-cloud-apis/user-tokens#create-a-personal-access-token) if you have developer, IT, or Read-only license.
```

[Create a service token](/docs/dbt-cloud-apis/service-tokens) with `Job Admin` privileges from within <Constant name="cloud" />. Ensure that you save a copy of the token, as you won’t be able to access this later.

> As an alternative, you can [create a personal account token if have a Developer license](https://docs.getdbt.com/docs/dbt-cloud-apis/user-tokens#create-a-personal-access-token).

Reviewer: ```suggestion
As an alternative, you can [create a PAT](/docs/dbt-cloud-apis/user-tokens#create-a-personal-access-token) if you have a Developer, IT, or Read-only license.
```
